src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2957,99 +2956,40 @@ Real64 GLHESlinky::calcHXResistance(EnergyPlusData &state)
 
 //******************************************************************************
 
-Real64 GLHEBase::interpGFunc(Real64 const LnTTsVal // The value of LN(t/TimeSS) that a g-function
-) const
+Real64 GLHEBase::interpGFunc(Real64 const LnTTsVal) const
```

Comment:
Let's see, this function is only about 100x easier to read and understand now.  This is great.  I might have a few suggested tweaks, but this seems awesome.